### PR TITLE
fix($stacked-area-chart): Add stream center option

### PR DESCRIPTION
**Pull request by jeznag**
Stream center option works better for datasets including zero

### CODE REVIEW

File: build/nv.d3.js [hunk 1]
```diff
@@ -8800,7 +8800,6 @@ nv.models.lineWithFocusPlusSecondaryChart = function() {
 
         viewFinder.width(availableWidth);
         viewFinder.margin(focusMargin);
-        viewFinder.xAxis.tickFormat(primaryXAxis.tickFormat());
```
Comment (jufemaiz) [inline build/nv.d3.js]: I'll ask dumb questions. Why removed?

File: build/nv.d3.js [hunk 2]
```diff
@@ -8918,7 +8917,7 @@ nv.models.lineWithFocusPlusSecondaryChart = function() {
 
         const currentExtent = d3.extent(
           d3.merge(seriesPrimary.concat(seriesSecondary)),
-          getX
```
Comment (jufemaiz) [inline build/nv.d3.js]: getX defined somewhere?

File: build/nv.d3.js [hunk 3]
```diff
@@ -8985,8 +8984,7 @@ nv.models.lineWithFocusPlusSecondaryChart = function() {
       chart.update = function() {
         const {
           dataSecondary,
-          dataPrimary,
-          seriesSecondary
```
Comment (jufemaiz) [inline build/nv.d3.js]: seriesSecondary - that's weird.

File: build/nv.d3.js (hunk 2 above)
Comment (jeznag) [inline build/nv.d3.js]: Came from https://github.com/COzero/nvd3/commit/57c5ea4365db388cc3e45b9801b3856409e68f63 but I neglected to rebuild before merging changes

File: build/nv.d3.js (hunk 3 above)
Comment (jeznag) [inline build/nv.d3.js]: Same https://github.com/COzero/nvd3/commit/57c5ea4365db388cc3e45b9801b3856409e68f63

File: build/nv.d3.js (hunk 1 above)
Comment (jeznag) [inline build/nv.d3.js]: https://github.com/COzero/nvd3/commit/bef22547cc602ac44702992a1f9bdb78c4658f6f

**Pull request merged**